libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -289,25 +322,30 @@ export class MultiInputComponent implements OnInit, ControlValueAccessor, OnChan
         }
 
         // Handle popover placement update
-        if (
+        if ((
```

Comment:
`if(shouldPopoverBeUpdated(previousLength,currentLenght))`. This is more readable